src/replication.c

```diff
@@ -1445,6 +1446,56 @@ send_replica_snapshot(spec_t *spec, replica_t *replica, uint64_t io_seq,
 	return ret;
 }
 
+static int
+send_replica_resize_operation(spec_t *spec, replica_t *replica, uint64_t io_seq,
+	zvol_op_code_t mgmt_opcode, size_t size, rcommon_mgmt_cmd_t *rcomm_mgmt) {
+	zvol_io_hdr_t *rmgmtio = NULL;
+	size_t data_len;
+	zvol_op_resize_data_t *resize_cmd;
+	void *data;
+	int ret = 0;
+	uint64_t num = 1;
+	mgmt_cmd_t *mgmt_cmd, *tmp_mgmt_cmd;
+
+	mgmt_cmd = malloc(sizeof(mgmt_cmd_t));
+	memset(mgmt_cmd, 0, sizeof(mgmt_cmd_t));
+	mgmt_cmd->rcomm_mgmt = rcomm_mgmt;
```

Comment:
who is freeing rcomm_mgmt? there seems to be a leak here.

Reply:
As of now we are passing `null` rcommon_mgmt_cmd_t so no need to release the memory.(might be extra code can we remove it?)

Reply:
yeah, do not even pass that argument. no need to that.